File: Teste_Carro/Carro.py
class Carro:
    def __init__(self):
        self.ano = None
        self.cor = None
        self.modelo = None
        self.marca = None

    def exibir(self):
        print("Ano do veiculo:", self.ano)
        print("Modelo do veiculo:", self.modelo)
        print("Cor do veiculo:", self.cor)
        print("Marca do veiculo:", self.marca)

    def ligar(self):
        print("O carro ligou")

carro1 = Carro()
carro1.ano = 2017
carro1.cor = "Preto"
carro1.marca = "Honda"
carro1.modelo = "Sedan"

carro1.exibir()
carro1.ligar()

File: Teste_Carro/Carro.java
package teste_POO;

public class Carro {
	int ano;
	String cor;
	String modelo;
	String marca;


void exibir() {
	System.out.println("Ano do veiculo: " +ano);
	System.out.println("Modelo do veiculo: "+modelo);
	System.out.println("Cor do veicuilo: "+cor);
	System.out.println("Marca do veiculo: "+marca);
}

void ligar() {
	System.out.println("O carro ligou");
}
}